zqd/handlers.go

```diff
@@ -85,6 +86,12 @@ func handlePacketSearch(c *Core, w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
+	pcapReader, err := pcapio.NewReader(slicer)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
```

Comment:
```suggestion
		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Reply:
I didn't think that was right but I copied it from the one above, which should also change right?

Reply:
Yes.